demos/coreHTTP/http_demo_s3_download_multithreaded.c

```diff
@@ -156,13 +158,27 @@
 /**
  * @brief The stack size to use for HTTP tasks.
  */
-#define httpexampleTASK_STACK_SIZE                        ( configMINIMAL_STACK_SIZE * 2 )
+#define httpexampleTASK_STACK_SIZE                        ( configMINIMAL_STACK_SIZE * 4 )
```

Comment:
How about making this configurable? Adding a #ifndef #endif check around it will help.